server/container/guice/cassandra-rabbitmq-ldap-guice/src/main/java/org/apache/james/CassandraRabbitMQLdapJamesServerMain.java

```diff
@@ -26,19 +26,12 @@
 import com.google.inject.Module;
 import com.google.inject.util.Modules;
 
-public class CassandraRabbitMQLdapJamesServerMain {
+public class CassandraRabbitMQLdapJamesServerMain implements JamesServerMain {
```

Comment:
Why do you need to implement an empty interface?

Reply:
I find it convenient to have a marker interface to find all james mains and not forget them in the future.

Also having a common method avoids 7 duplication.